### PR TITLE
chore: fix lint errors that will occur with new major versions (partial)

### PR DESCRIPTION
**Pull request by Robbert**

### CODE REVIEW

File: components/Accordion/src/index.tsx
Comment (Borai) [inline components/Accordion/src/index.tsx:1]: Do we need to fill those index file with exports of the particular component?

Comment (Robbert) [reply components/Accordion/src/index.tsx:1]: Yes, that is what is happening everywhere. The readme and one package.json (probably copied from the readme) was outdated, all other package.jsons were okay.

Comment (Borai) [reply components/Accordion/src/index.tsx:1]: So I this something we need to create a task for?

Comment (Robbert) [reply components/Accordion/src/index.tsx:1]: No, this was the last one.

**Pull request merged**